### ISSUE TITLE
Since 3.9.1 I do experience lags in internet radio stations

### ISSUE THREAD

**Issue by hannes101** [quodlibet/quodlibet#2445]
Hi all,
since the upgrade to 3.9.1 I do experience lagging sound on many different internet radio stations. For example I do have that issue with various stations and it's not dependent on the provider. Did anything change on the lower level what might be the culprit? It most probably isn't caused by the internet connection, since I am pretty much directly am connected to the backbone of the university.
This is quodlibet on windows, need to check if it's also on linux, when I am home.

**Comment by lazka** [comment]
What do you mean by lagging? Audio dropouts?

**Comment by hannes101** [comment]
Yeah, like a jump in a vinyl plate or a scratched CD, but since I increased the buffer time to 10s it's fine and much better.

**Comment by lazka** [comment]
Hm, maybe something changed with the GStreamer buffering strategy. I don't know much about the internals there.

**Comment by debianxfce** [comment]
I have the same bug. Good test material is a trance radio, with 5 seconds buffer songs do not play in tempo. With 10 seconds buffer quodlibet plays almost ok. Vlc plays ok.
http://norway.discovertrance.com:8000

**Comment by lazka** [comment]
@debianxfce Works fine here :/ Which distro/version?

**Comment by Humfred** [comment]
Same bug for me but with podcast (audio feed). I use windows 7 and  Quod Libet 4.0.2.
With Quod Libet 3.8.1 it's work fine.

**Comment by debianxfce** [comment]
Thanks for QuodLibet and its author to guide me to internet radios. I am using the Chromium web browser for internet radios. For example this is a good and well designed site:
https://www.internet-radio.com/stations/80s/

**Comment by CreamyCookie** [comment]
I don't know if it's related but I'm using 3.9.1 and have also noticed weird lags, but they happen for normal music playback (from the library). Mostly they seem to happen when I've had QL playing in the background and then bring the QL window back to the front (unminimizing it).

**Comment by CreamyCookie** [comment]
@hannes101 @debiangamer @Humfred Is this still an issue with Quod Libet 4.4.0?
